lib/matplotlib/tests/test_agg.py

```diff
@@ -244,3 +247,79 @@ def test_pil_kwargs_tiff():
     im = Image.open(buf)
     tags = {TiffTags.TAGS_V2[k].name: v for k, v in im.tag_v2.items()}
     assert tags["ImageDescription"] == "test image"
+
+
+@pytest.fixture
+def chunk_limit_setup():
+    N = 100_000
+    dpi = 500
+    w = 5*dpi
+    h = 6*dpi
+
+    # just fit in the width
+    x = np.linspace(0, w, N)
+    # and go top-to-bottom
+    y = np.ones(N) * h
+    y[::2] = 0
+
+    idt = IdentityTransform()
+    # make a renderer
+    ra = RendererAgg(w, h, dpi)
+    # setup the minimal gc to draw a line
+    gc = ra.new_gc()
+    gc.set_linewidth(1)
+    gc.set_foreground('r')
+    # make a Path
+    p = Path(np.vstack((x, y)).T)
+    # effectively disable path simplification (but leaving it "on")
+    p.simplify_threshold = 0
+
+    return ra, gc, p, idt
+
+
+def test_chunksize_hatch_fail(chunk_limit_setup):
+    ra, gc, p, idt = chunk_limit_setup
+
+    gc.set_hatch('/')
+
+    with pytest.raises(OverflowError, match='with hatch'):
+        ra.draw_path(gc, p, idt)
```

Comment:
Does it make sense to also test these with the suggested "fix" to make sure the fix works?  Or do we do that adequately elsewhere?

Reply:
We do not want to actually do that as this path takes significant amounts of time to draw if we allow it to split!